src/Domain/SizeOfElement.cpp

```diff
@@ -16,6 +16,47 @@
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/StdArrayHelpers.hpp"  // IWYU pragma: keep
 
+namespace {
+template <size_t VolumeDim>
+inline tnsr::I<double, VolumeDim, Frame::Logical> logical_face_center(
+    const size_t logical_dim, const Side& side) noexcept {
+  tnsr::I<double, VolumeDim, Frame::Logical> result{{{0.0}}};
+  result.get(logical_dim) = (side == Side::Lower ? -1.0 : 1.0);
+  return result;
+}
+
+template <size_t VolumeDim>
+inline double distance_between_face_centers(
+    const tnsr::I<double, VolumeDim, Frame::Inertial>& lower_center,
+    const tnsr::I<double, VolumeDim, Frame::Inertial>& upper_center) noexcept {
+  auto center_to_center = make_array<VolumeDim>(0.0);
+  for (size_t inertial_dim = 0; inertial_dim < VolumeDim; ++inertial_dim) {
+    center_to_center.at(inertial_dim) =
+        upper_center.get(inertial_dim) - lower_center.get(inertial_dim);
+  }
+  return magnitude(center_to_center);
+}
+}  // namespace
+
+template <size_t VolumeDim>
+std::array<double, VolumeDim> size_of_element(
+    const ElementMap<VolumeDim, Frame::Inertial>&
+        logical_to_inertial_map) noexcept {
+  auto result = make_array<VolumeDim>(0.0);
+  for (size_t logical_dim = 0; logical_dim < VolumeDim; ++logical_dim) {
```

Comment:
`logical_dim` -> `logical_index`

Reply:
Can you help me understand why this is an improvement? This is looping over {xi, eta, zeta}, which I think we've been calling the logical dimensions... (also in `direction.dimension()`)?

Reply:
We understand `logical_dim` ~ `VolumeDim` when there aren't symmetries. You are using this to index into tensor components. Does that make sense?